### DISCUSSION TITLE
Is it possible to disable Assistant from the .env file ?

### DISCUSSION THREAD

**Discussion by mikekuzak** [danny-avila/LibreChat#2446] [Help Wanted]
Leave ASSISTANTS_API_KEY= blank to disable this endpoint
Does not seem to work with version 0.7.0

**Comment by thiswillbeyourgithub** [comment]
~~Same here, I have no use for assistants and I can't figure out how to disable endpoints~~

**Reply by thiswillbeyourgithub** [reply]
Wait I just figured it out: I had left "=user_config" instead of setting it to "=". Now the endpooints I won't use are not appearing.

**Comment by danny-avila** [comment]
Try setting this list `ENDPOINTS=` https://www.librechat.ai/docs/configuration/dotenv#endpoints
